src/test/groovy/org/dormitory/autobotsoccub/engine/GameEngineShould.groovy

```diff
@@ -0,0 +1,87 @@
+package org.dormitory.autobotsoccub.engine
+
+import org.dormitory.autobotsoccub.engine.exception.GameDoesNotExistException
+import org.dormitory.autobotsoccub.engine.model.Game
+import org.dormitory.autobotsoccub.engine.model.GameData
+import spock.lang.Specification
+
+import static org.dormitory.autobotsoccub.engine.model.PlayPosition.GK
+
+class GameEngineShould extends Specification {
+
+    def GK_A = 0
+    def FW_A = 1
+    def GK_B = 2
+    def FW_B = 3
+
+    Game game
+    GameEngine sut = new GameEngine()
+
+    def setup() {
+        game = Game.builder()
+                .goalKeeperA(GK_A)
+                .forwardA(FW_A)
+                .goalKeeperB(GK_B)
+                .forwardB(FW_B)
+                .build()
+    }
+
+    def "successfully start new game"() {
+        expect:
+        sut.startGame(FW_A, game)
+        sut.stopGame(FW_A)
```

Comment:
you don't need actually to stop game, the same for test below

Reply:
I misunderstood you. Fixed